kubernetes/resource_kubernetes_runtime_class_v1_test.go

```diff
@@ -0,0 +1,142 @@
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	nodev1 "k8s.io/api/node/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestAccKubernetesruntime_class_v1_basic(t *testing.T) {
+	var conf nodev1.RuntimeClass
+	rcName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	resourceName := "kubernetes_runtime_class_v1.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		IDRefreshName:     resourceName,
+		IDRefreshIgnore:   []string{"metadata.0.resource_version"},
+		ProviderFactories: testAccProviderFactories,
+		//CheckDestroy:      testAccCheckKubernetesRuntimeClassDestroy,
+		Steps: []resource.TestStep{
+			//creating a run time class
+			{
+				Config: testAccKubernetesruntime_class_v1_basic(rcName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesruntime_class_v1Exists(resourceName, &conf),
+					resource.TestCheckResourceAttr(resourceName, "metadata.0.name", rcName),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"metadata.0.resource_version"},
+			},
+			{
+				Config: testAccKubernetesruntime_class_v1_addAnnotations(rcName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesruntime_class_v1Exists("kubernetes_runtime_class_v1.test", &conf),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.annotations.%", "2"),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.annotations.TestAnnotationOne", "one"),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.annotations.TestAnnotationTwo", "two"),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.labels.%", "0"),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.name", rcName),
+					resource.TestCheckResourceAttrSet("kubernetes_runtime_class_v1.test", "metadata.0.generation"),
+					resource.TestCheckResourceAttrSet("kkubernetes_runtime_class_v1.test", "metadata.0.resource_version"),
+					resource.TestCheckResourceAttrSet("kubernetes_runtime_class_v1.test", "metadata.0.uid"),
+				),
+			},
+			{
+				Config: testAccKubernetesruntime_class_v1_addLabels(rcName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesruntime_class_v1Exists("kubernetes_runtime_class_v1.test", &conf),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.annotations.%", "2"),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.annotations.TestAnnotationOne", "one"),
+					resource.TestCheckResourceAttr("kubernetes_runtime_class_v1.test", "metadata.0.annotations.Different", "1234"),
```

Comment:
In the test resource you have the second annotation `TestAnnotationTwo` with value `two`. Please update it here.